bridge/client/app/project-board/project-board.component.ts

```diff
@@ -22,6 +22,7 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
   public project$: Observable<Project>;
   public contextId: string;
   private _rootEventsTimerInterval = 30;
+  private readonly _projectTimerInterval = 30_000;
```

Comment:
Have never seen this syntax yet with `30_000`, but it seems to be working. Do you know of any browser restriction or is this feature supported by all "modern" browsers?

Reply:
Just found out that it is not supported by IE, but Edge and of course all other desktop browsers do have this feature already implemented.
https://caniuse.com/mdn-javascript_grammar_numeric_separators

Reply:
The separators are omitted on build

Reply:
Just created an issue to document supported browsers https://github.com/keptn/keptn/issues/4637
but I realized that it works, as you said because it is omitted on build.